rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
@kentcdodds I think we can do 
```js
Object.assign(
  glamorous,
  ThemeProvider,
)
```

import {Div, ThemeProvider} from 'glamorous'